chain/client/src/test_utils/test_env.rs

```diff
@@ -139,6 +139,26 @@ impl TestEnv {
         self.process_block(id, block.unwrap(), Provenance::PRODUCED);
     }
 
+    // Produces block by the client that is the block producer for the given height.
+    pub fn produce_block_simple(&mut self, height: BlockHeight) {
+        let client = &self.clients[0];
+        let tip = client.chain.head().unwrap();
+        let epoch_id = tip.epoch_id;
```

Comment:
It probably doesn't matter in this case, but I thought `epoch_id` had to be retrieved from `height` not from `tip`

Reply:
oh my good point, let me look into this